packages/bot-web-ui/src/stores/app-store.js

```diff
@@ -35,6 +54,13 @@ export default class AppStore {
         window.addEventListener('beforeunload', this.onBeforeUnload);
 
         main_content.getCachedActiveTab();
+
+        when(
```

Comment:
Don't need a reaction here?

Reply:
@balakrishna-deriv This reaction we need to fix the problem that occurs when you click to refresh the page, when the error message 'Deriv Bot is not available to customers from the EU' is gone.  The reason that in the `showDigitalOptionsMaltainvestError()` `client.is_eu_country`(from client store - core) we get **true** asynchronously. 
Details: 
1)in the constructor it always undefined 
2) if write block `if (client.is_eu_country){ showDigitalOptionsMaltainvestError() }` it also never come **true** inside `onMount()` 
3) so only helps `reaction when()` inside `onMount()` to get correct value of `is_eu_country` and doesn't allow the user to trade if If we forbid him